src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -112,14 +112,12 @@ public class AssertsFalseAttribute : Attribute
 
         protected static CSharpCompilationOptions WithNonNullTypesTrue(CSharpCompilationOptions options = null)
         {
-            return (options ?? TestOptions.ReleaseDll).WithSpecificDiagnosticOptions(
-                new[] { new System.Collections.Generic.KeyValuePair<string, ReportDiagnostic>("CS" + ((int)ErrorCode.WRN_PragmaNonNullTypes).ToString("0000"), ReportDiagnostic.Error) });
+            return (options ?? TestOptions.ReleaseDll).WithNullable(true);
         }
 
         protected static CSharpCompilationOptions WithNonNullTypesFalse(CSharpCompilationOptions options = null)
         {
-            return (options ?? TestOptions.ReleaseDll).WithSpecificDiagnosticOptions(
-                new[] { new System.Collections.Generic.KeyValuePair<string, ReportDiagnostic>("CS" + ((int)ErrorCode.WRN_PragmaNonNullTypes).ToString("0000"), ReportDiagnostic.Suppress) });
+            return (options ?? TestOptions.ReleaseDll).WithNullable(false);
```

Comment:
>WithNullable(false) [](start = 55, length = 19)

Do we have any tests that exercise the option `null`?

Reply:
We should change `Nullable` to a `bool` so we don't need to test that case.

---
In reply to: [224933444](https://github.com/dotnet/roslyn/pull/30479#discussion_r224933444) [](ancestors = 224933444)

Reply:
> Do we have any tests that exercise the option null?

Yes. Those are all the tests that didn't apply NonNullTypes attribute on a module.

---
In reply to: [224933817](https://github.com/dotnet/roslyn/pull/30479#discussion_r224933817) [](ancestors = 224933817,224933444)

Reply:
Any test that doesn't call `WithNonNullTypesTrue/False` will use `null`. So the case should be covered.
Closing

---
In reply to: [224933817](https://github.com/dotnet/roslyn/pull/30479#discussion_r224933817) [](ancestors = 224933817,224933444)